code/modules/integrated_electronics/core/tools.dm

```diff
@@ -273,7 +273,9 @@
 		/obj/item/weapon/tool/crowbar,
 		/obj/item/weapon/tool/screwdriver,
 		/obj/item/device/multitool,
-		/obj/item/device/integrated_electronics/wirer, /obj/item/device/integrated_electronics/debugger, /obj/item/device/integrated_electronics/detailer //CHOMP Edit. And had to put all of them on one line because 
+		/obj/item/device/integrated_electronics/wirer, //CHOMP Edit.
+		/obj/item/device/integrated_electronics/debugger, //CHOMP Edit.
+		/obj/item/device/integrated_electronics/detailer, //CHOMP Edit. 
```

Comment:
As I suggested in discord, use commas at the end of each line, even if the lines end with a comment. Travis doesn't like a list without commas at the end of each line that's followed by another part of said list.